packages/contracts-bedrock/contracts/L1/OptimismPortal.sol

```diff
@@ -185,6 +196,66 @@ contract OptimismPortal is Initializable, ResourceMetering, Semver {
             "OptimismPortal: invalid withdrawal inclusion proof"
         );
 
+        // Designate the withdrawalHash as proven by storing the `outputRoot`, `timestamp`,
+        // and `l2BlockNumber` in the `provenWithdrawals` mapping. A certain withdrawal
+        // can be proved multiple times and thus overwrite a previously stored `ProvenWithdrawal`,
+        // but this is safe due to the replay check in `finalizeWithdrawalTransaction`.
+        provenWithdrawals[withdrawalHash] = ProvenWithdrawal({
+            outputRoot: outputRoot,
+            timestamp: uint128(block.timestamp),
+            l2BlockNumber: uint128(_l2BlockNumber)
+        });
+
+        // Emit a `WithdrawalProven` event.
```

Comment:
Including an indexed `_tx.from` and `_tx.to` in this event might be helpful for offchain services

Reply:
Added 👍